shared/stories/prop-providers.js

```diff
@@ -21,11 +21,10 @@ const defaultFollowers = ['max', 'akalin']
 const Usernames = (following: string[] = defaultFollowing, you: string = defaultYou) => ({
   Usernames: (ownProps: _Usernames.ConnectedProps): _Usernames.Props => {
     const {usernames} = ownProps
-    const users = (usernames || []).map(username => ({
-      username,
-      following: following.includes(username),
-      you: username === you,
-    }))
+    const users = (usernames || [])
+      .map(username => ({username, following: following.includes(username), you: username === you}))
+      .filter(u => !ownProps.skipSelf || !u.you)
```

Comment:
Added this filter line to work with the chat conversation header, since mergeProps in `common-adapters/usernames.js` had it.

Reply:
ConnectedUsernames is the only component that has a skipSelf prop so I don't think so. It also didn't affect any of the existing storyshots.